### ISSUE TITLE
Feature request: Offset

### ISSUE THREAD

**Issue by jdamboeck** [Smirnov75/ComfyUI-mxToolkit#3]
![image](https://github.com/Smirnov75/ComfyUI-mxToolkit/assets/20957463/54e85346-6080-4aee-8c2a-c2885dc5a5f3)

If there was the possibility of setting an offset by dragging the box the 2D slider would make an awesome tool for cropping.

**Comment by Smirnov75** [comment]
Yes, it's a good idea, but it might unnecessarily complicate this node. I'll think about it. Most likely, it would be better to create a separate node for these purposes

**Comment by PrometheusDante** [comment]
Love this node pack first of all! Going to include in in my published workflows from now on. The world must know of this sleek beauty and convenience! :D
And I was just about to suggest the same thing. 
The way I'm currently "optimizing" the cropping procedure in my workflows is looking really ridiculous, by going through `CR Image Border` `ImageDrawRectangle` and `Cut By Mask`. It works perfectly fine, couldn't find a better method, but still, it just feels hacky.

An extra node similar to _Impact Pack_'s `Preview Bridge` with image input and two dragable points to mark a rectangular area across the image and then outputting the crop would be beautiful and an elegant solution to the problem.

Thank you very much for your time and efforts!
